### DISCUSSION TITLE
Welcome to istio Discussions!

### DISCUSSION THREAD

**Discussion by morlay** [querycap/istio#23] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jli3388** [comment]
@morlay I am trying to follow your instruction to deploy istio onto a pi cluster and did ran into problems. 
From what I can tell it seems it can't find a proper node, saw your note to set "spec.components.*.k8s.affinity". Where exactly do I set it?  I can't seem to find a similar spec section u have there in the README.md. 

Is this a param I need to give during istioctl install so it overrides all affinity setting?

**Reply by morlay** [reply]
@jli3388 since 1.10, this setting is not need any more.
